HW28/tests/test_auth.py

```diff
@@ -0,0 +1,11 @@
+from loguru import logger
+
+
+def test_login_positive(get_auth_token_valid):
+    assert get_auth_token_valid is not None
+    logger.info(f"The auth token: {get_auth_token_valid}")
```

Comment:
What do you think to call method in test, so you can pass any payload?
```suggestion
def test_login_positive():
    auth_payload_valid = {
    ....
    }
    asd = get_token(auth_payload_valid)
   assert ....
    logger.info(f"The auth token: {get_auth_token_valid}")
```